### DISCUSSION TITLE
import nc file in grass gis

### DISCUSSION THREAD

**Discussion by MHaghiri** [OSGeo/grass#3751] [Q&A] [answered]
Hi,
I want to import nc file in grass gis with this code (r.import input=/path/to/ file/name.nc output=name) on windows system.
but I got the error.
in grass have any tools to import the nc file like raster file or not?
or  have another way to import nc file in grass?
could you provide me a way to import it?
best.

**Comment by ninsbl** [comment, marked as answer]
Please provide the Error message you get. NetCDF files often have subdatasets, so you have to choose data you want to import...
See e.g. the HDF examples here: https://grass.osgeo.org/grass83/manuals/r.in.gdal.html
Netcdf examples could be added to the manual...
If your NetCDF adheres to the CF convention, you could try: https://grass.osgeo.org/grass82/manuals//addons/t.rast.import.netcdf.html

**Comment by ninsbl** [comment]
I take the liberty to convert this issue to a discussion. When you provide an error message you get we can open a more specific bug ticket (if we identify a bug) or a ticket for improved documentation...
